index.html

```diff
@@ -67,17 +67,34 @@
 				the primary method of interchange or consumption.</p>
 		</section>
 		<section id="sotd">
-			<p>Many details of this draft are under active consideration within the Publishing Working Group and are
-				subject to change. The most prominent known issues have been identified in this document and links
-				provided to comment on them.</p>
-
-			<p>The most significant change to this draft is the splitting of the manifest from the requirements for Web
-				Publications so that it can serve as a model for packaged formats that desire a slimmer set of default
-				requirements.</p>
-
-			<p>The Publishing Working Group is actively producing proofs of concept of Web Publications and the
-				technologies required to process them. For a list of these activities, please refer to the <a
-					href="https://w3c.github.io/wpub/experiments/index.html">experiments page</a>.</p>
+			<p>Due to the lack of practical business cases for Web Publications, and the consequent lack of commitment
+				to implement the technology, the Publishing Working Group has chosen to publish this document as a Note
```

Comment:
Let us add a link to the WG. Respec does it automatically in the status section, but somewhat later...